### PR TITLE
Create Readme

### PR DESCRIPTION
**Pull request by haroldsphinx**
- Create Readme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+https://registry.terraform.io/providers/RedisLabs/rediscloud/latest
+
+## What does this do?
+
+Creates all resources (Subscription, Clusters, Databases and VPC peering) needed to provison a redis enterprise cluster 
+
+## How to use this module?
+
+```hcl
+module "confluent-kafka-cluster" {
```
Comment (darron) [inline README.md:14]: `kafka`?

File: redis.tf [hunk 2]
```diff
@@ -35,21 +35,21 @@ resource "rediscloud_subscription" "subscription" {
...
+  gcp_network_name = var.network_name
+}
+
+resource "google_compute_network_peering" "gcp-network-peering" {
+  name         = var.network_peering_name
+  network      = var.network_name
+  peer_network = format(
+      "https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s",
+      rediscloud_subscription_peering.gcp-project-peering.gcp_project_id, 
+      rediscloud_subscription_peering.gcp-project-peering.gcp_redis_network_name
```
Comment (darron) [inline redis.tf:51]: Does this work automatically? I thought it didn't export those values based on the discussions here:

https://github.com/RedisLabs/terraform-provider-rediscloud/issues/141

**Review by darron — COMMENTED** [review]
A couple comments.

File: redis.tf (hunk 2 above)
Comment (haroldsphinx) [reply redis.tf:48]: You are right, it didnt work, I uncommneted it to test create the nba-staging redis, what works is manually inputing the network and project id

**Pull request merged**